src/sqlfluff/rules/L002.py

```diff
@@ -41,6 +41,7 @@ class Rule_L002(BaseRule):
 
     """
 
+    groups = ("all", "core")
     config_keywords = ["tab_space_size"]
```

Comment:
Why is group a Tuple, and `config_keywords` a List?

Should `groups` be a List?